### PR TITLE
IBLI on the Ground

### PR DESCRIPTION
**Pull request by NofarGizra**
#63
#94

### CODE REVIEW

**Comment by NofarGizra** [comment]
Update:
- **Currently in progress:** Support migration of "IBLI on the Ground" pages.
- **Still ToDo:** Create a view for "IBLI on the Ground".

**Comment by NofarGizra** [comment]
Update:
IBLI on the Ground is now displaying nodes from nodequeue.
**Still ToDo:** Make it a ctools plugin.

**Comment by NofarGizra** [comment]
onthegrounddd

**Comment by niryariv** [comment]
we need the same width for all images, as well as height. feel free to just crop them if needed.

**Comment by NofarGizra** [comment]
Update:
- Now it is a Ctools plugin.

Still ToDo:
- Use image styles to scale & crop the images.

**Comment by NofarGizra** [comment]
Update:
- All images are now the same size (263x157 px).

**Comment by NofarGizra** [comment]
groundground

File: ibli/modules/custom/ibli_general/plugins/content_types/ibli_on_the_ground/ibli-on-the-ground.tpl.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        <span class="title"><?php print t('IBLI ON THE GROUND'); ?></span><span class="decoration"></span><span class="decoration"></span><span class="decoration"></span>
+      </h2>
+    </div>
+  </div>
+  <div class="row">
+    <?php foreach ($nodequeue as $page): ?>
+      <div class="col-md-3 col-sm-6">
+        <div class="thumbnail">
+          <a href="<?php print $page['url']; ?>"><?php print $page['image']; ?></a>
+          <div class="visit"><a href="#"><i class="fa fa-question-circle"></i> <?php print t('More details...'); ?></a></div>
```
Comment (amitaibu) [inline ibli/modules/custom/ibli_general/plugins/content_types/ibli_on_the_ground/ibli-on-the-ground.tpl.php]: add link to the node.

File: ibli/modules/custom/ibli_general/plugins/content_types/ibli_on_the_ground/ibli_on_the_ground.inc [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    $nodes = node_load_multiple(array_keys($result));
+
+    foreach ($nodes as $node) {
+      $wrapper = entity_metadata_wrapper('node', $node);
+
+      // Get data for image style.
+      $image = $wrapper->field_image->value();
+      $image_vars = array(
+        'path' => $image['uri'],
+        'alt' => $node->title,
```
Comment (amitaibu) [inline ibli/modules/custom/ibli_general/plugins/content_types/ibli_on_the_ground/ibli_on_the_ground.inc]: check_plain().

File: ibli/modules/custom/ibli_general/plugins/content_types/ibli_on_the_ground/ibli_on_the_ground.inc [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+  // Load "IBLI On The Ground" nodequeue data.
+  $query = db_select('nodequeue_queue', 'nq');
+  $query->innerJoin('nodequeue_nodes', 'nn', 'nq.qid = nn.qid');
+  $result = $query
+    ->fields('nn', array('nid'))
+    ->condition('nq.name', 'ibli_on_the_ground')
+    ->execute()
+    ->fetchAllAssoc('nid');
+  if (!empty($result)) {
```
Comment (amitaibu) [inline ibli/modules/custom/ibli_general/plugins/content_types/ibli_on_the_ground/ibli_on_the_ground.inc]: Return early.

File: ibli/modules/custom/ibli_general/plugins/content_types/ibli_on_the_ground/ibli_on_the_ground.inc [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+function ibli_general_ibli_on_the_ground_content_type_render($subtype, $conf, $args, $context) {
+  $variables = array(
+    'nodequeue' => array(),
+  );
+
+  // Load "IBLI On The Ground" nodequeue data.
+  $query = db_select('nodequeue_queue', 'nq');
+  $query->innerJoin('nodequeue_nodes', 'nn', 'nq.qid = nn.qid');
+  $result = $query
+    ->fields('nn', array('nid'))
```
Comment (amitaibu) [inline ibli/modules/custom/ibli_general/plugins/content_types/ibli_on_the_ground/ibli_on_the_ground.inc:22]: Check node is published.

**Comment by NofarGizra** [comment]
Merged.

**Pull request closed without merging**